guide/13-managing-arcgis-applications/hub-premium-guide.ipynb

```diff
@@ -0,0 +1,471 @@
+{
```

Comment:
<p>ArcGIS Hub supports engagement through in-person and virtual events. Events are meetings for people to support an Initiative and are scheduled by an organizer and have many attendees. An Event will have an associated Group that will allow you to include content for preparation as well as gather and archive content during the event for later retrieval or analysis.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1448/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>